### ISSUE TITLE
Not working as expected

### ISSUE THREAD

**Issue by mercurymotos** [MegaphoneJon/nosmsurltracking#1]
Very excited about this extension! But it is not working as expected...
Installed onto a site with 4.7.7, using Twiliio.  The SMS preview shows the tracking URL and an entry for the URL is created in the database.

I upgraded to 4.7.10, same result.

Am I missing something?  Or is the extension not working in 4.7?

**Comment by MegaphoneJon** [comment]
I haven't tested the extension in 4.7 - I wrote this for a single client, who has yet to upgrade.  They're on a support plan that means that I WILL be paid at some point to look into this - but not for a couple of months.  Sorry!

**Comment by mercurymotos** [comment]
Fair enough!  Maybe the info.xml file shouldn't affirm compatibility with 4.7 for now?

**Comment by MegaphoneJon** [comment]
Argh, did I do that?  In that case I probably developed on a 4.7 test machine.  Let me test it and get back to you.
